code/_onclick/hud/hud_datum.dm

```diff
@@ -170,6 +170,15 @@
 			if(infodisplay.len)
 				mymob.client.screen -= infodisplay
 
+		if(HUD_STYLE_ACTIONHUD)	//No HUD
```

Comment:
Currently, you can toggle to this, this will need a line change on 122, and probably a new define for the max HUDs the user can switch through

Reply:
Yeah, its a bit awkward, because you currently press the button while on NOHUD, and nothing seems to change (at least if you dont have action buttons)